pkg/resources/pool_stub_test.go

```diff
@@ -27,7 +27,7 @@ var _ = Describe("PoolStub", func() {
 		fs = &utils.FakeFilesystem{
 			Dirs: []string{
 				"sys/bus/pci/devices/0000:00:00.1/net/enp2s0f1",
-				"sys/bus/pci/devices/0000:00:00.1/physfn/net/enp2s0f0",
+				"sys/bus/pci/devices/0000:01:00.0/net/enp2s0f0",
```

Comment:
nit: doesnt really affect the test, but should this point to a different netdev than the one on L#31 ?